docs/CONTRIBUTING.md

```diff
@@ -51,15 +51,17 @@ application icon if built on Linux or Windows due to non-trival icon
 generation on these platforms. In order to develop the source code you will
 need the following setup:
 
-- **The Rust Toolchain (nightly-2019-11-04)**
+- **The Rust Toolchain**
 
-  This project uses several features available only in the nightly Rust
-  toolchain.  Please use the
-  [the Rust toolchain installer](https://rustup.rs) to install it:
+  If you don't have Rust installed yet, you can use the 
+  [the Rust toolchain installer](https://rustup.rs) to install it. This 
+  project uses several features available only in the nightly Rust
+  toolchain. The exact version required is specified through the 
+  `rust-toolchain` file in the repository root and the correct version will be 
+  installed by cargo automatically. To install all other required components 
+  run the following commands:
```

Comment:
I read about rust-toolchain file at https://github.com/rust-lang/rustup#the-toolchain-file so I suspect this is a purely rustup feature. So it's not like you _can_ install rust using rustup, but you _have to_ install rust with rustup if you want to have build scripts using the proper toolchain automatically.

That matters e.g. on Manjaro (and Archlinux too), where you have two separate packages: rust for install latest stable rust as a system package, and rustup which will mantain many rust toolchains.

Reply:
Ah okay. So that limits the usefulness somewhat. But using the toolchain file would stil allow us to omit the `override` command from the setup at least.